core/controllers/resources.py

```diff
@@ -95,7 +95,8 @@ def get(self, page_context, page_identifier, asset_type, encoded_filename):
                     page_context == feconf.ENTITY_TYPE_EXPLORATION or
                     page_context == feconf.ENTITY_TYPE_SKILL or
                     page_context == feconf.ENTITY_TYPE_TOPIC or
-                    page_context == feconf.ENTITY_TYPE_STORY):
+                    page_context == feconf.ENTITY_TYPE_STORY or
+                    page_context == feconf.ENTITY_TYPE_VOICEOVER_APPLICATION):
```

Comment:
It might make sense to create something like `VALID_ENTITY_TYPES` in `feconf` and just do `page_context in feconf.VALID_ENTITY_TYPES` here.